packages/iTwinUI-react/src/core/Alert/Alert.tsx

```diff
@@ -96,14 +90,14 @@ export const Alert = (props: AlertProps) => {
       </span>
 
       {onClose && (
-        <IconButton
-          styleType='borderless'
-          size='small'
+        <button
           onClick={onClose}
           aria-label='Close'
+          type='button'
+          className='iui-alert-button'
         >
-          <SvgCloseSmall aria-hidden />
-        </IconButton>
+          <SvgCloseSmall aria-hidden className='iui-alert-button-icon' />
```

Comment:
Why did this file change? Shouldn't it be handled in #753?

Reply:
It is handled there. I don't know why github sees this change. It does not show up in VS code PR 
![image](https://user-images.githubusercontent.com/83585998/181429855-6c150924-f85d-4c38-8e74-0b258ff340c1.png)

Reply:
I'd say branch is missing a merge with v1

Reply:
Fixed